### PR TITLE
Add F9 continue key to ESIL

### PR DESCRIPTION
**Pull request by sylvainpelissier**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**
<kbd>F9</kbd> key was to bind to continue if no debugger were attached. Now it will perform the `aec` command for ESIL.

**Test plan**

I could not find how to simulate the press of <kbd>F9</kbd> key in a test.

### CODE REVIEW

File: libr/core/visual.c [hunk 1]
```diff
@@ -836,7 +836,13 @@ static void visual_breakpoint(RCore *core) {
 }
 
 static void visual_continue(RCore *core) {
-	r_core_cmd (core, "dc", 0);
+	if (r_config_get_i (core->config, "cfg.debug")) {
+		r_core_cmd (core, "dc", 0);
+	}
```
Comment (radare) [inline libr/core/visual.c:841]: Else goes here

File: libr/core/visual.c [hunk 2]
```diff
@@ -836,7 +836,13 @@ static void visual_breakpoint(RCore *core) {
...
+	else {
+		r_core_cmd (core, "aec", 0);
```
Comment (radare) [inline libr/core/visual.c:843]: Use ;.ar* in the same line

**Comment by sylvainpelissier** [comment]
Ok done.

**Comment by sylvainpelissier** [comment]
Do you have an idea to test it ?

**Comment by radare** [comment]
we dont test visual modes

**Pull request merged**